packages/components/color-picker/src/use-color-slider.ts

```diff
@@ -335,6 +335,25 @@ export const useColorSlider = ({
       const n = thumbPercent
       const { width: w } = thumbSize ?? { width: 0 }
 
+      const hslValue = `hsl(${value}, 100%, 50%)`
+      const colorName = (() => {
```

Comment:
Do not define functions here. This would cause the function to be redefined each time it is executed.

You should define `getColorName` in `packages/utils/src/color.ts` to be generic.

Reply:
Fixed to define functions in `packages/utils/src/color.ts`.